tee-supplicant/src/tee_supplicant.c

```diff
@@ -481,9 +485,14 @@ static int get_dev_fd(uint32_t *gen_caps)
 
 static int usage(int status)
 {
-	fprintf(stderr, "Usage: tee-supplicant [-d] [<device-name>]\n");
-	fprintf(stderr, "       -d: run as a daemon (fork after successful "
+	fprintf(stderr, "Usage: tee-supplicant [options] [<device-name>]\n");
+	fprintf(stderr, "\t-h, --help: this help\n");
+	fprintf(stderr, "\t-d, --daemonize: run as a daemon (fork after successful "
 			"initialization)\n");
+	fprintf(stderr, "\t-f, --fs-parent-path: secure fs parent path\n");
+	fprintf(stderr, "\t-t, --ta-dir: TAs dirname under %s\n", TEEC_LOAD_PATH);
+	fprintf(stderr, "\t-p, --plugin-path: plugin load path\n");
```

Comment:
I suggest adding the default values in brackets:
```suggestion
	fprintf(stderr, "Usage: tee-supplicant [options] [<device-name>]\n");
	fprintf(stderr, "\t-h, --help: this help\n");
	fprintf(stderr, "\t-d, --daemonize: run as a daemon (fork after successful "
			"initialization)\n");
	fprintf(stderr, "\t-f, --fs-parent-path: secure fs parent path [%s]\n",
			supplicant_params.fs_parent_path);
	fprintf(stderr, "\t-t, --ta-dir: TAs dirname under %s [%s]\n", TEEC_LOAD_PATH,
			supplicant_params.ta_dir);
	fprintf(stderr, "\t-p, --plugin-path: plugin load path [%s]\n",
			supplicant_params.plugin_load_path);
```

Reply:
updated as per suggestion